go/ephemeral/ephemeral_test.go

```diff
@@ -16,15 +16,44 @@ func TestNewDeviceEK(t *testing.T) {
 	_, err := kbtest.CreateAndSignupFakeUser("t", tc.G)
 	require.NoError(t, err)
 
-	metadata, err := PublishNewDeviceEK(context.Background(), tc.G)
+	publishedMetadata, err := PublishNewDeviceEK(context.Background(), tc.G)
 	require.NoError(t, err)
 
 	fetchedDevices, err := GetOwnDeviceEKs(context.Background(), tc.G)
 	require.NoError(t, err)
 
 	require.Equal(t, 1, len(fetchedDevices))
-	require.Equal(t, metadata, fetchedDevices[0])
-	require.Equal(t, 1, metadata.Generation)
+	for _, fetchedDevice := range fetchedDevices {
+		require.Equal(t, publishedMetadata, fetchedDevice)
+	}
+	require.Equal(t, 1, publishedMetadata.Generation)
+}
+
+func TestNewUserEK(t *testing.T) {
+	tc := libkb.SetupTest(t, "ephemeral", 2)
+	defer tc.Cleanup()
+
+	fakeUser, err := kbtest.CreateAndSignupFakeUser("t", tc.G)
+	require.NoError(t, err)
+
+	// The test user has a PUK, but it's not automatically loaded. We have to
+	// explicitly sync it.
+	keyring, err := tc.G.GetPerUserKeyring()
+	require.NoError(t, err)
+	err = keyring.Sync(context.Background())
+	require.NoError(t, err)
+
+	_, err = PublishNewDeviceEK(context.Background(), tc.G)
```

Comment:
let's also do a second device here so when we publish the `UserEK` it boxes for both